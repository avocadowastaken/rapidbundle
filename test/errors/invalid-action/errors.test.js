import { runErrorTest } from "../../utils/runTest.js";

runErrorTest(import.meta.url);
