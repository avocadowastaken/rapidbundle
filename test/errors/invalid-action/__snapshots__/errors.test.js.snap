// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`invalid-action 1`] = `
[HH:MM:SS] Resolving build manifests [started]
[HH:MM:SS] → Checking 'package.json'
[HH:MM:SS] → Checking 'action.yml'
[HH:MM:SS] Resolving build manifests [failed]
[HH:MM:SS] → Invalid 'action.yml':
  .runs.using: Invalid enum value. Expected 'node12' | 'node14' | 'node16', received 'node'
  .runs.main: Expected to be in the 'dist' directory, received 'main.js'
`;
